german/schwimmen/hunger/momentmal/verstehen/verstehen.md

```diff
@@ -0,0 +1 @@
+Der Leader der Männer hat dich mit Finger gezeigt, dann gleich eine Krone hoch gehoben und hingekniet. Sie wollen dich als ihr neuer König auf dem Thron sehen!!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.